catalog/imaging/imaging-transformer-chipping/src/test/java/org/codice/alliance/imaging/chip/transformer/CatalogOutputAdapterTest.java

```diff
@@ -96,8 +165,803 @@ public void testGetBinaryContent() throws IOException, MimeTypeParseException {
+        });
+
+        chipNitfSegmentFlow.forEachImageSegment(imageSegment1 -> {
+
+            assertThat(imageSegment1.getImageCoordinates()
+                    .getCoordinate00()
+                    .getLongitude(), closeTo(22, 0.01));
+            assertThat(imageSegment1.getImageCoordinates()
+                    .getCoordinate00()
+                    .getLatitude(), closeTo(18, 0.01));
+
+            assertThat(imageSegment1.getImageCoordinates()
+                    .getCoordinate0MaxCol()
+                    .getLongitude(), closeTo(38, 0.01));
+            assertThat(imageSegment1.getImageCoordinates()
+                    .getCoordinate0MaxCol()
+                    .getLatitude(), closeTo(18, 0.01));
+
+            assertThat(imageSegment1.getImageCoordinates()
+                    .getCoordinateMaxRowMaxCol()
+                    .getLongitude(), closeTo(38, 0.01));
+            assertThat(imageSegment1.getImageCoordinates()
+                    .getCoordinateMaxRowMaxCol()
+                    .getLatitude(), closeTo(2, 0.01));
+
+            assertThat(imageSegment1.getImageCoordinates()
+                    .getCoordinateMaxRow0()
+                    .getLongitude(), closeTo(22, 0.01));
+            assertThat(imageSegment1.getImageCoordinates()
+                    .getCoordinateMaxRow0()
+                    .getLatitude(), closeTo(2, 0.01));
+
+            assertThat(imageSegment1.getNumberOfColumns(), is((long) chipWidth));
+            assertThat(imageSegment1.getNumberOfRows(), is((long) chipHeight));
+            assertThat(imageSegment1.getImageCategory(), is(ImageCategory.UNKNOWN));
+            assertThat(imageSegment1.getImageComments(),
+                    is(Collections.singletonList("MyComment")));
+            assertThat(imageSegment1.getIdentifier(), is("0123456789"));
+            assertThat(imageSegment1.getImageIdentifier2(), is("abc"));
+            assertThat(imageSegment1.getImageMagnification(), is("1.0 "));
+            try {
+                assertThat(imageSegment1.getImageTargetId()
+                        .textValue(), is(new TargetId("                 ").textValue()));
+            } catch (NitfFormatException e) {
+                fail(e.getMessage());
+            }
+            assertThat(imageSegment1.getImageSource(), is(""));
+            assertThat(imageSegment1.getImageDateTime()
+                    .getSourceString(), is(dateTime.getSourceString()));
+            assertThat(imageSegment1.getImageRepresentation(),
+                    is(ImageRepresentation.RGBTRUECOLOUR));
+
+        });
+
+    }
+
+    private ImageCoordinates getImageCoordinates() throws NitfFormatException {
+        return new ImageCoordinates(new ImageCoordinatePair[] {icp("200000N0200000E"),
+                icp("200000N0400000E"), icp("000000N0400000E"), icp("000000N0200000E")});
+    }
+
+    private ImageSegment getImageSegment(long originalWidth, long originalHeight, DateTime dateTime,
```

Comment:
Having these private methods below all the test methods might be better from a readability standpoint